csharp_dotnetcore/Using-Cards/Using_Cards/Startup.cs

```diff
@@ -0,0 +1,126 @@
+                    // IStorage dataStore = new FileStorage(System.IO.Path.GetTempPath());
+
+                    // For production bots use the Azure Table Store, Azure Blob, or 
+                    // Azure CosmosDB storage provides, as seen below. To include any of 
+                    // the Azure based storage providers, add the Microsoft.Bot.Builder.Azure 
+                    // Nuget package to your solution. That package is found at:
+                    //      https://www.nuget.org/packages/Microsoft.Bot.Builder.Azure/
+
+                    // IStorage dataStore = new Microsoft.Bot.Builder.Azure.AzureTableStorage("AzureTablesConnectionString", "TableName");
+                    // IStorage dataStore = new Microsoft.Bot.Builder.Azure.AzureBlobStorage("AzureBlobConnectionString", "containerName");
+
+                    var convoState = new ConversationState(dataStore);
+                    options.State.Add(convoState);
+
+                    // Add State to BotStateSet Middleware (that require auto-save)
+                    // The BotStateSet Middleware forces state storage to auto-save when the Bot is complete processing the message.
+                    // Note: Developers may choose not to add all the State providers to this Middleware if save is not required.
+                    var stateSet = new BotStateSet(options.State.ToArray());
+                    options.Middleware.Add(stateSet);
+                });
+
+                services.AddSingleton<CardsBotAccessors>(sp =>
+                {
+                    var options = sp.GetRequiredService<IOptions<BotFrameworkOptions>>().Value;
+                    if (options == null)
+                    {
+                        throw new InvalidOperationException(
+                            "BotFrameworkOptions must be configured prior to setting up the State Accessors");
+                    }
+
+                    var conversationState = options.State.OfType<ConversationState>().FirstOrDefault();
+
+                    if (conversationState == null)
+                    {
+                        throw new InvalidOperationException(
+                            "ConversationState must be defined and added before adding conversation-scoped state accessors.");
+                    }
+
+                    // Create Custom State Property Accessors
+                    // State Property Accessors enable components to read and write individual properties, without having to 
+                    // pass the entire State object.
+                    var accessors = new CardsBotAccessors
+                    {
+                        ConversationDialogState =
+                            conversationState.CreateProperty<DialogState>(CardsBotAccessors.DialogStateName),
+                    };
+
+                    return accessors;
+                });
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e);
```

Comment:
This isn't a console bot. 

Remove the whole try/catch.

Reply:
Must have just been an oversight on my part.